src/Components/QuestionList/QuestionCard/QuestionCard.tsx

```diff
@@ -29,6 +40,14 @@ export const QuestionCard = ({ question, showDelete }: CardProps) => {
                         Answered
                     </Button>
                     <QuestionTags tags={tags} />
+                    <Button
+                        variant="outline-secondary"
+                        onClick={() => {
+                            navigate(editQuestionPath[Paths.questionEdit](id));
+                        }}
+                    >
+                        ✎
+                    </Button>
```

Comment:
its kinda confusing to have the button here on the card, should it not be a mode like delete? mainly because the correct answer has the same look as the edit button so it's not very intuitive, its better than the button on the side but still not the best solution.